modules/host/persist.go

```diff
@@ -3,39 +3,42 @@ package host
 import (
 	"path/filepath"
 
-	"github.com/NebulousLabs/Sia/encoding"
 	"github.com/NebulousLabs/Sia/modules"
+	"github.com/NebulousLabs/Sia/persist"
 )
 
+var persistMetadata = persist.Metadata{
+	Header:  "Sia Host",
+	Version: "0.3.3",
```

Comment:
One thing to note about these versions is that they don't need to keep parity with the Sia version.

Though, it probably does make sense to have the version be in-line with a Sia release. That makes it easier to go back and figure out where things changed later.